biosys/apps/main/models.py

```diff
@@ -1027,3 +1028,12 @@ def has_destroy_permission(request):
 
     def has_object_destroy_permission(self, request):
         return is_admin(request.user) or self.is_data_engineer(request.user)
+
+
+class CustomUserManager(UserManager):
```

Comment:
Just a note: you cannot change the django User model in the middle of a project unless you go through some painstaking data migration and check that the new user model is properly used by views etc...  
Unfortunately Biosys didn't implement its own user model from day one so I think we have to deal with the current default User model (+ Profile)

Reply:
ok, i'm pretty sure this is only so it would've worked with django admin as well anyway, does that even matter? should i just put it into the signup bit?

Reply:
signup yes